contrib/kube-prometheus/README.md

```diff
@@ -153,31 +164,62 @@ A hidden `_config` field is located at the top level of the object this library
         },
 
         prometheus+:: {
+            names: 'k8s',
```

Comment:
there is also 'namespaces' option here.. 
Can this list of "config options" - be pulled somehow.. so one could build a "complete config file" - with all options.. and then adjust from there (it would show all options one has - in a very userfriendly way).. ?